src/components/applications/ApplicationDetails.js

```diff
@@ -47,14 +50,53 @@ export const ApplicationDetails = ({application, isHR, reload}) => {
     }
 
     const nextStage = () => {
+        if ( nextStageRequiresDevs() ) {
+            const devMails = []
+            Swal.fire({
+                title: "We need more info",
+                icon: "warning",
+                text: 'Next stage of this appliaction requires people with technical knowledge. Please, insert their mails here, splitting them with commas (,)',
+                input: "text",
+                preConfirm: (devsMails) => {
+                    if (devsMails) {
+                        devsMails.split(",")
+                            .map(mail => mail.trim())
+                            .forEach(mail => devMails.push(mail))
+                    } else {
+                        Swal.showValidationMessage("Please, add mails of such people, as you won't be able to change it later.")
+                    }
+                }
+            }).then(result => {
+                if (result.isConfirmed) {
+                    doSetNextStage(devMails)
+                }
+            })
```

Comment:
Nie robimy jakiegoś checka czy to poprawne maile?

Reply:
pewnie się powinno XDD